packages/cli/subtasks/pick-function.js

```diff
@@ -0,0 +1,45 @@
+const { subtask } = require('hardhat/config');
+const inquirer = require('inquirer');
+const autocomplete = require('inquirer-list-search-prompt');
+
+const { SUBTASK_PICK_FUNCTION } = require('../task-names');
+
+subtask(SUBTASK_PICK_FUNCTION, 'Pick a function from the given contract').setAction(
+  async (taskArguments, hre) => {
+    const abi = hre.deployer.deployment.abis[hre.cli.contract];
+    return await _prompt(abi);
+  }
+);
+
+async function _prompt(abi) {
```

Comment:
Review if this can be abstracted and re-used between the `pick-contract` and `pick-function` subtasks.

Reply:
Implemented

Reply:
Done